apps/health_campaign_field_worker_app/lib/pages/beneficiary/deliver_intervention.dart

```diff
@@ -87,11 +85,6 @@ class _DeliverInterventionPageState
                       form: () => buildForm(context),
                       builder: (context, form, child) {
                         return ScrollableContent(
-                          header: const Column(children: [
-                            BackNavigationHelpHeaderWidget(
-                              showcaseButton: ShowcaseButton(),
```

Comment:
don't remove the complete one, add showBackNavigation: false, similar to home screen